website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "JDBC"
+id: sl-jdbc
+description: "Integrate and use the JDBC API to query your metrics."
+tags: [Semantic Layer, APIs]
+---
+
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+</VersionBlock>
+
+The dbt Semantic Layer Java Database Connectivity (JDBC) API enables users to query metrics and dimensions using the JDBC protocol, while also providing standard metadata functionality. 
+
+A JDBC driver is a software component enabling a Java application to interact with a data platform. Here's some more information about our JDBC API:
+
+- The Semantic Layer JDBC API utilizes the open-source JDBC driver with ArrowFlight SQL protocol.
+- You can download the JDBC driver from [Maven](https://search.maven.org/remotecontent?filepath=org/apache/arrow/flight-sql-jdbc-driver/12.0.0/flight-sql-jdbc-driver-12.0.0.jar). 
+- The dbt Semantic Layer supports ArrowFlight SQL driver version 12.0.0 and higher. 
+- You can embed the driver into your application stack as needed, and you can use dbt Labs' [example project](https://github.com/dbt-labs/example-semantic-layer-clients) for reference.
+- If you’re a partner or user building a homegrown application, you’ll need to install an AWS root CA to the Java Trust [documentation](https://www.amazontrust.com/repository/) (specific to Java and JDBC call).
+
+Partners of dbt Labs can use the JDBC API to build integrations in their tools with the dbt Semantic Layer
+
+
+## Using the JDBC API
+
+If you are a dbt user or partner with access to dbt Cloud and the Semantic Layer, you can setup and test this API with data from your own instance by configuring the Semantic Layer and obtaining the right JDBC connection parameters described in this document. (PROVIDE link to set up?). 
```

Comment:
@mirnawong1  same thing here. giving partners the ability to follow the guide for setup in case they want to test these apis out on their own is probably useful